test/configCases/wasm/arraybuffer-import-object/index.js

```diff
@@ -0,0 +1,6 @@
+it("should work", () => import("./module.wat").then(({ bar }) => {
+    const bytes = new Uint8Array(bar.buffer, 0, 2);
+    const util = require("util");
+    const string = new util.TextDecoder("utf8").decode(bytes);
```

Comment:
I added this because `TextDecoder` is only available as a global from Node.js 11.0.0 onward, and this ensures compatibility with Node.js 10.24.1. Let me know if there's a better approach!